homu/server.py

```diff
@@ -29,6 +29,10 @@ def index():
 
 @get('/queue/<repo_label>')
 def queue(repo_label):
+    logger = g.logger.getChild('queue')
+
+    logger.debug('repo_label: %s', repo_label)
```

Comment:
I didn't know that `.debug()` allowed variadic arguments! However, as we're primarily using the `.format()`-based text formatting, it would be better to keep it consistent.